src/ImageGallery.jsx

```diff
@@ -652,16 +654,17 @@ export default class ImageGallery extends React.Component {
     const CENTER = 'center';
     const RIGHT = 'right';
 
-    switch (index) {
-      case (currentIndex - 1):
-        alignment = ` ${LEFT}`;
-        break;
-      case (currentIndex):
-        alignment = ` ${CENTER}`;
-        break;
-      case (currentIndex + 1):
-        alignment = ` ${RIGHT}`;
-        break;
+    if (index === currentIndex - 1) {
+      alignment = ` ${LEFT}`;
+    } else if (index === currentIndex) {
+      alignment = ` ${CENTER}`;
+    } else if (index === currentIndex + 1) {
+      alignment = ` ${RIGHT}`;
+    } else if (this.props.numberOfImagesToPreLoad > 0) {
```

Comment:
could we leave the switch case as is and wrap in if else 

```
if (this.props.numberOfImagesToPreload) {
  ...
} else {
  switch (index) {
    case()
  }
}
```

i find switch case easier to read and it helps diffing changes to original since this is a fork